db/seeds.rb

```diff
@@ -1,5 +1,6 @@
 League.delete_all
-
+Team.delete_all
```

Comment:
You don't need to delete the models in the `seeds` file. When you run the `rails db:setup` task it will automatically drop, create and seed the database.